src/components/Map/USACountyMap.js

```diff
@@ -91,45 +88,41 @@ const USACountyMap = React.memo(
               <Geographies geography={geoStates}>
                 {({ geographies }) =>
                   geographies
-                    .filter(geo => isStateFips(geo.id))
+                    .filter(geo => stateFipsCodes.includes(geo.id))
                     .map(geo => {
-                      const { name } = geo.properties;
                       const fipsCode = geo.id;
-                      const stateCode = getStateCode(name);
-                      const stateUrl = `/${getCanonicalUrl(fipsCode)}`;
-                      const locationName = getLocationNameForFips(fipsCode);
-
+                      const state = regions.findByFipsCode(fipsCode);
                       // Using a custom SVG to place the northern mariana islands to increase
                       // accessibility due to the small size.
-                      if (stateCode === 'MP') {
+                      if (state.stateCode === 'MP') {
                         return (
                           <Link
-                            key={stateCode}
-                            to={stateUrl}
-                            aria-label={locationName}
+                            key={state.stateCode}
+                            to={state.relativeUrl}
+                            aria-label={state.fullName}
                           >
                             <MarianaIslands
                               key={geo.rsmKey}
-                              onMouseEnter={() => setTooltipContent(name)}
+                              onMouseEnter={() => setTooltipContent(state.name)}
```

Comment:
Nit: Here's a chance where maybe using `state.fullName` makes sense? I still find `name` to be a bit ambiguous

Reply:
for states is the same thing, but I prefer to use `fullName`, is more explicit